web/ui/src/lib/clustersMap/campus/seoul.ts

```diff
@@ -9,7 +9,7 @@ export class Seoul extends Campus implements ICampus {
   name = (): CampusNames => 'seoul';
 
   extractorRegexp = (): RegExp =>
-    /(?<clusterWithLetter>c(?<cluster>\d+))(?<rowWithLetter>r(?<row>\d+))(?<workspaceWithLetter>s(?<workspace>\d+))/i;
+    /(?<clusterWithLetter>c(?<cluster>[\d\w]+))(?<rowWithLetter>r(?<row>\d+))(?<workspaceWithLetter>s(?<workspace>\d+))/i;
```

Comment:
This regexp is invalid, I fixed it due to the fact the `cluster` group must be without number (and don't integrate `x`). Move the catcher inside the `clusterWithLetter` group and change the anchor `\w` to `x` due to the fact the anchor `\w` takes the range `[a-zA-Z0-9]`, so `c10r2p1` doesn't works fine with `\w` (`clusterWithLetter` responds with `c1` and `cluster` responds with `0`).

```suggestion
    /(?<clusterWithLetter>cx?(?<cluster>[\d]+))(?<rowWithLetter>r(?<row>\d+))(?<workspaceWithLetter>s(?<workspace>\d+))/i;
```

Sorry for the late review, I takes somes days off :)